modules/core/src/v2/coins/eos.ts

```diff
@@ -217,4 +218,31 @@ export class Eos extends BaseCoin {
       throw new Error(`address validation failure: ${addressDetails.address} vs ${rootAddressDetails.address}`);
     }
   }
+
+  /**
+   * Assemble keychain and half-sign prebuilt transaction
+   *
+   * @param params
+   * @param params.txPrebuild {Object} prebuild object returned by platform
+   * @param params.prv {String} user prv
+   */
+  signTransaction(params) {
+    const prv = params.prv;
+    const txData = params.txPrebuild.rawTx;
+    const tx = params.txPrebuild.tx;
+
+    const signBuffer = Buffer.from(txData, 'hex');
+    const privateKeyBuffer = utxoLib.HDNode.fromBase58(prv).getKey().getPrivateKeyBuffer();
+    const signature = ecc.Signature.sign(signBuffer, privateKeyBuffer).toString();
+
+    tx.signatures.push(signature);
+
+    const txParams = {
+      transaction: tx,
+      txHex: txData,
```

Comment:
shouldnt the txHex be 
```
tx.buildIncomplete().serialize()
```
or something? looks like we're just returning the unsigned txhex

Reply:
no, this is already serialized signing data. The thing that we send is `transaction`, which is just json lol